scratch3/style.js

```diff
@@ -1,6 +1,7 @@
 import SVG from "./draw.js"
 import cssContent from "./style.css.js"
 
+var Style
 export default Style = {
```

Comment:
Would an un-named default export be better? i.e. `export default {`

Reply:
This is technically an unnamed default export. `Style = {}` is just a variable assignment expression - so this assigns the variable and returns the same value which is used by `export default`.

Variable assignment is necessary because some of the codes in the same file references `Style` and it'll raise a syntax error due to missing variables.

This should be replaced with a class in the future, but it's out-of-scope for this PR.